### DISCUSSION TITLE
Trouble pushing to implementation branch

### DISCUSSION THREAD

**Discussion by DariBerrie** [DevSkillsHQ/discusssions#13] [Help] [answered]
[solved] I cloned from the standard template instead of my exercise!

**Comment by fenske** [comment, marked as answer]
Hi @DariBerrie. I'm Anton from DevSkills. It seems you are pushing to one of the boilerplate repositories rather than the repo with your task definition.

To fix this, locate the repository with your task definition (you can find it through the email you originally received from us), and push your changes from there.

Feel free to re-open this discussion if it doesn't work.

Best,
Anton.

**Comment by DariBerrie** [comment]
I don't know how I realized it before I got your message, but thank you! I
had one too many tabs open, I think 😅

**Reply by fenske** [reply]
Np at all. I'm glad it got resolved. 

I wish you the best of luck with the rest of your interview process!
